gradle.properties

```diff
@@ -25,7 +25,7 @@ org.gradle.caching=true
 
 mavenGroupId         = com.microsoft.kiota
 mavenMajorVersion = 0
```

Comment:
this change is breaking, we should be bumping the major version IMHO

Reply:
no, we're still not stable for the Java language, so breaking changes are still permitted as per our support policy